staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -403,6 +406,33 @@ func RecordRequestAbort(req *http.Request, requestInfo *request.RequestInfo) {
 	requestAbortsTotal.WithContext(req.Context()).WithLabelValues(reportedVerb, group, version, resource, subresource, scope).Inc()
 }
 
+// RecordDroppedRequest records that the request was rejected via http.TooManyRequests.
+func RecordDroppedRequest(req *http.Request, requestInfo *request.RequestInfo, component string, isMutatingRequest bool) {
```

Comment:
I like the change but without a test, I'm worried something can go wrong. Like we're using `WithLabelValues` which means this is going to be additive to the most important metric we have, and if any of the labels are misplaced, then we've effectively destroyed the utility of our most valuable metric.